docs/faq.md

```diff
@@ -139,6 +139,10 @@ If you want to change the behavior of the mounts, that will also require [additi
 New-NetFirewallRule -DisplayName "WSL" -Direction Inbound -InterfaceAlias "vEthernet (WSL)" -Action Allow
 ```
 
+#### **Q: Can I map `host.docker.internal` or `host.rancher-desktop.internal` to `host-gateway` with the flag `--add-host`?
+
+**A:** No. The special value `host-gateway` is something Docker Desktop specific and is not supported in Rancher Desktop yet. However, you can access services running on the host machine from within a container via `host.docker.internal` or `host.rancher-desktop.internal` without passing the `--add-host` flag.
```

Comment:
Just a tiny suggestion regarding the rewording:

No, the special value `host-gateway` is specific to Docker Desktop and is not yet supported in Rancher Desktop. However, you can access services running on the host machine from within a container using `host.docker.internal`or `host.rancher-desktop.internal` without passing the `--add-host` flag.

Reply:
@Nino-K , I applied your recommendation.